Goobi/plugins/opac/PicaPlugin/org/goobi/production/plugin/CataloguePlugin/PicaPlugin/MapRule.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+package org.goobi.production.plugin.CataloguePlugin.PicaPlugin;
+
+import org.apache.commons.configuration.HierarchicalConfiguration;
+
+/**
+ * Map rules are used to map data from norm data records to a media record. This
+ * is used inside a {@code ResolveRule}. Example usage:
+ *
+ * <pre>
+ * &lt;catalogue title="…">
```

Comment:
It is more readable then a XML encoding but even still horrible in my eyes. If I look into [Sun/Oracle documentation](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html) and [complex examples](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#examples) I don't see there any of encoding tags. Maybe this is a some strange behaviour of some IDE's which made this kind of documentation non-readable for plain-text users.